examples/hosts/app-integration/external-controller/package.json

```diff
@@ -22,6 +22,7 @@
     "lint:fix": "npm run eslint:fix",
     "prepack": "npm run webpack",
     "start": "webpack-dev-server",
+    "start:frs": "webpack-dev-server --define process.env.USE_FRS='\"true\"'",
```

Comment:
nit: consider making the env variable something like `FLUID_CLIENT` with values `frs` and `tinylicious` the default.

Reply:
Updated env variable to be `FLUID_CLIENT` which is set to `frs` for the `start:frs` command